src/udev/udev-builtin-net_setup_link.c

```diff
@@ -51,10 +51,8 @@ static int builtin_net_setup_link(struct udev_device *dev, int argc, char **argv
         }
 
         r = link_config_apply(ctx, link, dev, &name);
-        if (r < 0) {
-                log_error_errno(r, "Could not apply link config to %s: %m", udev_device_get_sysname(dev));
-                return EXIT_FAILURE;
-        }
+        if (r < 0)
+                log_warning_errno(r, "Could not apply link config to %s: %m", udev_device_get_sysname(dev));
```

Comment:
if we ignore such errors, then we should say so, i.e. this should be "Could not apply link config to %s, ignoring: %m" or so